spec/lib/bamboozled/employee_spec.rb

```diff
@@ -112,5 +113,31 @@
       expect(location).to eq "https://api.bamboohr.com/api/gateway.php/alphasights/v1/employees/44259"
     end
   end
+
+  describe "#update" do
+    it 'updates an employee in BambooHR' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.